test/src/test/java/jenkins/security/stapler/Security914Test.java

```diff
@@ -45,12 +46,11 @@
     public JenkinsRule j = new JenkinsRule();
     
     @Test
+    @WithPlugin("credentials.hpi")
```

Comment:
We no longer bundle `credentials` as a detached plugin as of commit commit c6f061f26a, which this test wanted. Fortunately, there's a `credentials.hpi` in the source tree, so we can get the same effect with a `@WithPlugin` annotation.